x-pack/legacy/plugins/siem/public/components/ml/create_descriptions_list.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiText, EuiSpacer, EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { Anomaly, NarrowDateRange } from './types';
+import { getScoreString } from './get_score_string';
+import { PreferenceFormattedDate } from '../formatted_date';
+import { createInfluencers } from './create_influencers';
+import { DescriptionList } from '../../../common/utility_types';
+import * as i18n from './translations';
+import { createExplorerLink } from './create_explorer_link';
+
+const LargeScore = styled(EuiText)`
+  font-size: 45px;
+  font-weight: lighter;
+`;
+
+export const createDescriptionsList = (
+  score: Anomaly,
+  startDate: number,
+  endDate: number,
+  interval: string,
+  narrowDateRange: NarrowDateRange
+): DescriptionList[] => {
+  const descriptionList: DescriptionList[] = [
+    {
+      title: i18n.MAX_ANOMALY_SCORE,
+      description: (
+        <>
+          <EuiSpacer size="m" />
+          <LargeScore>{getScoreString(score.severity)}</LargeScore>
+        </>
+      ),
+    },
+    {
+      title: (
+        <>
+          <EuiSpacer size="m" />
+          {i18n.ANOMALY_JOB}
+        </>
+      ),
+      description: (
+        <EuiFlexGroup direction="column" gutterSize="none" responsive={false}>
+          <EuiFlexItem grow={false}>{score.jobId}</EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiLink href={createExplorerLink(score, startDate, endDate)} target="_blank">
+              {i18n.VIEW_IN_MACHINE_LEARNING}
+            </EuiLink>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ),
+    },
+    {
+      title: i18n.DETECTED,
+      description: (
+        <EuiFlexGroup direction="column" gutterSize="none" responsive={false}>
+          <EuiFlexItem grow={false}>
+            <PreferenceFormattedDate value={new Date(score.time)} />
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiLink
+              onClick={() => {
+                narrowDateRange(score, interval);
+              }}
+              target="_blank"
+            >
+              {i18n.NARROW_TO_THIS_DATE_RANGE}
+            </EuiLink>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      ),
+    },
+    {
+      title: i18n.TOP_ANOMALY_SUSPECT,
+      description: (
+        <EuiFlexGroup direction="column" gutterSize="none" responsive={false}>
+          <EuiFlexItem grow={false}>{`${score.entityName}: “${score.entityValue}”`}</EuiFlexItem>
```

Comment:
Did you want [curly quotes](https://practicaltypography.com/straight-and-curly-quotes.html) here?

Reply:
That website looks like it says I do which I followed. If we aren't using those, then I can change them back.